README.md

```diff
@@ -70,14 +70,22 @@ below for details.
 
```

Comment:
Is the `How Does the BuildPack Work?` section above still accurate w.r.t. the use of profile.d? Or does the use of the new manifest change that workflow?

Reply:
This is still correct. I explored the ability to automatically install scripts, but the documentation was severely lacking and I wasn't sure how to do it correctly without significant work, sadly.